arangod/RocksDBEngine/RocksDBTransactionCollection.cpp

```diff
@@ -317,33 +325,35 @@ void RocksDBTransactionCollection::commitCounts(uint64_t trxId,
   // Update the collection count
   int64_t const adjustment = _numInserts - _numRemoves;
   if (commitSeq != 0) { // is '0' for filling new indexes
-    if (_numInserts != 0 || _numRemoves != 0 || _revision != 0) {
+    if (hasOperations()) {
```

Comment:
the previous code was only checking for inserts and removes, but `hasOperations()` will also check for updates:
```
122 bool RocksDBTransactionCollection::hasOperations() const {
123   return (_numInserts > 0 || _numRemoves > 0 || _numUpdates > 0);
124 }
```
Intentional?

Reply:
Furthermore, we are not explicitly checking `_revision` anymore here, just the number of modifications.